code/game/mecha/combat/gorilla.dm

```diff
@@ -0,0 +1,197 @@
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/cannon
+	name = "8.8cm KwK 47"
+	desc = "<i>Precision German engineering!</i>" // Why would you ever take this off the mech, anyway?
+	icon_state = "mecha_uac2"
+	equip_cooldown = 55 // 5.5 seconds
+	projectile = /obj/item/projectile/bullet/cannon
+	fire_sound = 'sound/weapons/cannon.ogg'
+	projectiles = 1
+	projectile_energy_cost = 1000
+	salvageable = 0 // We don't want players ripping this off a dead mech. Could potentially be a prize for beating it if Devs bless me and someone offers a nerf idea.
```

Comment:
If there's a suggestion for a nerf, please poke me; I think this'd be a nice prize if you _did_ somehow beat the god-damn Gorilla, but I also don't want you taking this god-weapon.

Reply:
Perhaps you could make it drop a somewhat nerfed version on salvage? The mech itself would have the un-nerfed engine, but when you salvage the wreckage it would be slower, pack less of a punch, etc.

Reply:
Given it causes 1000 damage and takes 1000 energy by default, I'm not sure how much either should be adjusted.